portfolio/src/main/client/src/components/sections/comments.ts

```diff
@@ -299,9 +300,38 @@ const Comment = (comment: CommentData): HTMLElement => {
     });
   }
 
-
   const readmore = ReadMore(new Text(comment.content));
-  readmore.root.classList.add('comment-content');
+
+  const btnsEl = htmlElement`
+    <div class="comment-controls">
+      ${readmore.root}
+    </div>`;
+
+  if ('contentLang' in comment && comment.contentLang in languages) {
```

Comment:
nit: you're safe to just write:
```
if (comment.contentLang in languages) {
```

You don't need the first part since `undefined in [undefined]` returns false anyways.

Reply:
This `if` statement is as much to convince TypeScript that this is safe as it is to convince me that this is safe. If I don't assert that `'contentLang' in comment`, then TypeScript won't let me access the properties of `TranslatedCommentData`.